bigbluebutton-html5/imports/ui/core/hooks/useMuteSoundAlert.ts

```diff
@@ -0,0 +1,60 @@
+import { useEffect, useRef } from 'react';
+import Auth from '/imports/ui/services/auth';
+import audioManager from '/imports/ui/services/audio-manager';
+import { getSettingsSingletonInstance } from '/imports/ui/services/settings';
+import useVoiceActivity from './useVoiceActivity';
+import logger from '/imports/startup/client/logger';
+import AudioService from '/imports/ui/components/audio/service';
+
+const MUTE_SOUND_PATH = '/resources/sounds/conference-muted.mp3';
+const UNMUTE_SOUND_PATH = '/resources/sounds/conference-unmuted.mp3';
+
+const useMuteSoundAlert = () => {
+  const currentUserId = Auth.userID;
+  const { data: voiceActivityStream } = useVoiceActivity();
+
+  const currentUserVoiceState = voiceActivityStream
+    ?.slice()
+    .reverse()
+    .find((activity) => activity.userId === currentUserId);
+  const isMuted = currentUserVoiceState?.muted;
```

Comment:
This can likely be substituted by a simpler and more efficient alternative via AudioManager.

```suggestion
  const isMuted = useReactiveVar(AudioManager._isMuted.value) as boolean;
```